plugins/network-elements/nuage-vsp/src/com/cloud/agent/api/element/ApplyStaticNatVspAnswer.java

```diff
@@ -31,4 +31,13 @@ public ApplyStaticNatVspAnswer(ApplyStaticNatVspCommand cmd, Exception e) {
         super(cmd, e);
     }
 
+    @Override
+    public boolean equals(Object obj) {
+        return super.equals(obj);
+    }
+
+    @Override
+    public int hashCode() {
+        return super.hashCode();
+    }
```

Comment:
Why does this class exist?  It adds no state or behavior to the ``Answer`` base class.  Why not simply use ``Answer`` in this circumstance?